transitions/add_info_document.js

```diff
@@ -0,0 +1,77 @@
+var infoDocId = function(id) {
+  return id + '-info';
+};
+
+var getSisterInfoDoc = function(db, docId, callback) {
+  db.medic.get(infoDocId(docId), function(err, body) {
+    if (err && err.statusCode !== 404) {
+      callback(err);
+    } else {
+      callback(null, body);
+    }
+  });
+};
+
+var generateInfoDocFromAuditTrail = function(audit, docId, callback) {
```

Comment:
I think it'd be cleaner if this function just got the created date, either create.timestamp or 'unknown'. There's a bit of duplication with filling in the other doc fields.